packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -297,7 +321,9 @@ export async function* OpenAIChatModel(
       e.tag == FunctionCall ||
       e.tag == FunctionResponse,
   });
-  yield AI.AppendOnlyStream;
+  if (!props.experimental_streamFunctionCallOnly) {
+    yield AI.AppendOnlyStream;
```

Comment:
Should we skip the `yield delta.content` on line 425 if this is set?

Reply:
Yes, good point.